modules/tensor_mechanics/src/userobjects/RambergOsgoodHardening.C

```diff
@@ -0,0 +1,31 @@
+#include "RambergOsgoodHardening.h"
+
+template<>
+InputParameters validParams<RambergOsgoodHardening>()
+{
+  InputParameters params = validParams<TensorMechanicsHardeningModel>();
+  params.addRequiredParam<Real>("yield_stress", "The yield stress value");
+  params.addRequiredParam<Real>("hardening_exponent", "The hardening exponent value");
+  params.addRequiredParam<Real>("reference_plastic_strain", "Reference plastic strain value");
+  return params;
+}
+
+RambergOsgoodHardening::RambergOsgoodHardening(const std::string & name, InputParameters parameters) :
+    TensorMechanicsHardeningModel(name, parameters),
+  _yield_stress(getParam<Real>("yield_stress")),
```

Comment:
All initializer lists (in this and the other files of this patch) need to be indented